docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1,6 +1,8 @@
 [#configuration]
 == Configuration
 
+=== SDK service configuration
```

Comment:
Changes in this PR is not included: https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3825/files

Reply:
Changes in **articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-key-vault.md** are not included.